pylint/checkers/typecheck.py

```diff
@@ -952,6 +945,24 @@ def visit_attribute(self, node):
             ):
                 continue
 
+            # ignore pattern added to generated-members
+            ignored = False
+            for pattern in self.config.generated_members:
```

Comment:
Rather than breaking out of the loop, and assuming the changes from the previous comment, this can simply be:
```suggestion
            for pattern in self.compiled_generated_members:
                  if pattern.match("%s.%s" % (owner.pytype(), node.attrname))):
                         return
```

There's no need to keep looping if a single inferred value matches the pattern.